### ISSUE TITLE
Error stack broken in node.js v0.12

### ISSUE THREAD

**Issue by tonila** [dasilvacontin/subclass-error#2]
I'm Using the following sample app:

```
var SubclassError = require('subclass-error'); 
var ClientError = SubclassError ("ClientError", {statusCode:400}); 

function hungry () {
    throw new ClientError ("wow, much forbidden, very subclass");
} 
hungry();
```

This is the result:

```
D:\projects\subclass-error>node app.js

D:\projects\subclass-error\app.js:6
    throw new ClientError ("wow, much forbidden, very subclass");
          ^
ClientError
    at SubclassError (D:\projects\subclass-error\node_modules\subclass-error\subclass-error.js:26:16)
    at Object.<anonymous> (D:\projects\subclass-error\app.js:2:19)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

The error stack is completely different from case, where new Error ("wow, much forbidden, very subclass"); is thrown:

```
D:\projects\subclass-error>node app.js
D:\projects\subclass-error\app.js:6
    throw new Error ("wow, much forbidden, very subclass");
          ^
Error: wow, much forbidden, very subclass
    at hungry (D:\projects\subclass-error\app.js:6:11)
    at Object.<anonymous> (D:\projects\subclass-error\app.js:8:1)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

**Comment by dasilvacontin** [comment]
Thanks for reporting! I'll check this out asap.

**Comment by dasilvacontin** [comment]
I've added some tests to the repo, and the stack is in fact broken in both `0.11` and `0.12`.

**Comment by dasilvacontin** [comment]
Solved via #4, thanks!
